config.yaml

```diff
@@ -1 +1,7 @@
-options: {}
+options:
+  application-config:
+    type: string
+    description: >
+      Spring Boot configuration properties in JSON format, it will be passed to 
```

Comment:
s/to/as?

Reply:
I rephrase the config description, thanks!